.github/workflows/ansible-test.yml

```
@@ -41,42 +44,19 @@ jobs:
           - devel
     runs-on: ubuntu-latest
     steps:
-
-      # ansible-test requires the collection to be in a directory in the form
-      # .../ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}/
-
-      - name: Check out code
-        uses: actions/checkout@v3
-        with:
-          path: ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
-
-      - name: Set up Python
-        uses: actions/setup-python@v3
+      # Run `ansible-test sanity` inside of Docker.
```

`ansible-test sanity` is not run inside the container, but the tests started by ansible-test are run inside a container.
```suggestion
      # Run sanity tests inside a Docker container.
```